docs/_plugins/abp-usermap-mybb.md

```diff
@@ -0,0 +1,12 @@
+---
+name: ABP Usermap MyBB
+category: 3rd-party-integration
+repo: https://community.mybb.com/mods.php?action=view&pid=1238
+author: CrazyCat
+author-url: https://gitlab.com/AnoBug
+demo: 
+compatible-v0: false
```

Comment:
Best to leave the `compatible` fields empty for the time being. Can be checked and changed on a one-by-one basis afterwards.

Reply:
I set only `compatible-v0` to empty and went through all plugins to check if some of them are only compatible with v0.